clang/lib/Sema/CheckExprLifetime.cpp

```diff
@@ -1093,6 +1093,87 @@ static bool pathOnlyHandlesGslPointer(const IndirectLocalPath &Path) {
   }
   return false;
 }
+// Result of analyzing the Path for GSLPointer.
+enum AnalysisResult {
```

Comment:
Nit: not sure what is the current policy about `enum class` vs `enum`. I think the first one might usually be preferred in new code, that being said this is scoped to this file so I don't have a strong opinion about it.

Reply:
We don’t have a strict policy on using enum class vs enum, it's typically left to the author's judgment.